index.html

```diff
@@ -380,6 +380,9 @@ <h3 class="text header-3">Testimonials</h3>
         <a class="text body" href="">Contact us</a>
       </div>
       <div class="subscribe">
+        <p class="text subtitle">Subscribe to our news</p>
```

Comment:
create div and place here: p, input and button
+ create style margin-bottom: 150px;
+ remove it from input